config/betamocks/services_config.yml

```diff
@@ -901,6 +900,13 @@
       :cache_multiple_responses:
         :uid_location: "url"
         :uid_locator: "\/services/claims/v2/veterans\/(?:.+)\/claims\/(.+)/5103"
+    # ClaimsAPI BD
```

Comment:
we should probably just generalize this as `Benefits Documents API`. It's not specific to Claims API, anyone can use this.